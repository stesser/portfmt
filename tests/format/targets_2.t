
foo:
	alsdjflaskj

	lksdjflsadjf

	saldjflajf

VAR=
<<<<<<<<<
foo:
	alsdjflaskj

	lksdjflsadjf

	saldjflajf

VAR=
