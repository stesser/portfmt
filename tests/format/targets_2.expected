foo:
	alsdjflaskj

	lksdjflsadjf

	saldjflajf

VAR=
